### ISSUE TITLE
slicing = None

### ISSUE THREAD

**Issue by cbontoiu** [openPMD/openPMD-viewer#278]
I know that in the past ```slicing=None``` used to be possible as I can read "When passing slicing=None, get_field returns a full 3D Cartesian array. This can be useful for further analysis by hand, with numpy (e.g. calculating the total energy in the field)."

I use a routine to compute the total charge of the electrons within the simulation domain such as 
```
tc = 0.0; 
for k in z:
eChargeDensity = ts.get_field(iteration = it, field = 'e_chargeDensity', 
     slice_across = 'z', slice_relative_position = -1 + k*percUZ);
    for j in y:
        for i in x:
              tc += uXspace*uYspace*uZspace*eChargeDensity[0][j, i];
     
        chargeElectrons.append("{0:.6f}".format(tc*1e12)); # in pC
        numberElectrons.append(str(int(-tc/e))); 
```
moving the Z-cutplane from -1 to +1 but this is quite slow and in the end I always get about 5k particles missing out of ~300k, as compared with an analytical calculation. I was wondering  if one can use now ```slice_relative_position = None``` and I could just skip the loop on z, but trying this gave me a small number of particles (~3k) which means that only a single cutplane was caught, not the whole 3D simulation domain. Is there anything I can do here?

Thank you

**Comment by RemiLehe** [comment]
@cbontoiu 
Thanks for opening this issue.
In addition to passing `slice_relative_position=None`, could you also try passing `slice_across = None` (or not pass `slice_across` at all)? This should do what you need I think.

Also, instead of writing explicit for loops, you could simply use `sum()`. In other words, I would suggest:

```python
tc = 0.0; 
eChargeDensity, info = ts.get_field(iteration = it, field = 'e_chargeDensity');
charge = info.dx * info.dy * info.dz * eChargeDensity.sum();
```

**Comment by cbontoiu** [comment]
> ```python
>  info.dx * info.dy * info.dz * eChargeDensity.sum();
> ```

Great! It works as you said and saves me about 20 mins as compared with the loops version.
Thank you so much.
Cristian
